src/mixins/date-mixin.ts

```diff
@@ -188,17 +188,15 @@ export default class DateMixin extends Mixins(CommonMixin) {
   /**
    * Converts an API datetime string (in UTC) to a date and time string (Month Day, Year at HH:MM am/pm
    * Pacific time).
-   * @example "2021-01-01T00:00:00.000000+00:00" -> "Dec 31, 2020 at 04:00 pm Pacific time" (PST example)
-   * @example "2021-07-01T00:00:00.000000+00:00" -> "Jun 30, 2021 at 05:00 pm Pacific time" (PDT example)
+   * @example "2021-01-01T00:00:00.000000+00:00" -> "December 31, 2020 at 04:00 pm Pacific time" (PST example)
+   * @example "2021-07-01T00:00:00.000000+00:00" -> "June 30, 2021 at 05:00 pm Pacific time" (PDT example)
    */
   apiToPacificDateTime (dateTimeString: string): string {
     if (!dateTimeString) return null // safety check
 
     const date = this.apiToDate(dateTimeString)
-    const dateStr = this.dateToPacificDate(date)
-    const timeStr = this.dateToPacificTime(date)
 
-    return `${dateStr} at ${timeStr} Pacific time`
+    return this.dateToPacificDateTime(date)
```

Comment:
The result of this method is now consistent with other UIs and various outputs, eg:

![image](https://user-images.githubusercontent.com/10002889/179128241-e911ec10-fa63-4ea2-bbcc-4463042fa737.png)